packages/react/src/progress-indicator/ProgressIndicatorItem.tsx

```diff
@@ -106,22 +103,22 @@ export const allProgressStatuses: Array<ProgressIndicatorItemStatus> = [
 
 type ProgressIndicatorItemProps = PropsWithChildren<{
 	as: ElementType;
-	background?: ProgressIndicatorBackground;
-	status: ProgressIndicatorItemStatus;
 	isActive: boolean;
+	status: ProgressIndicatorItemStatus;
+	background?: ProgressIndicatorBackground;
+	levelTwoItem?: ProgressIndicatorItem['levelTwoItem'];
```

Comment:
This doesn't feel like the right thing to do. It might get annoying having to manipulate the original object to add this key periodically.

I would have thought we'd have a `child`/`children` key which can always be present, and that has an `isActive` key that makes it visible.

I do wonder if `children` is better in case a form step could have 2 separate "edit" steps

Reply:
I can't think of a legitimate use case for `children`. I see it as if there are 2 separate edit steps, that would require 2 separate progress indicator pages.

`children` also implies that you can see a full sub list of nav items under the progress indicator item. That isn't what it is for.

I prefer it to be called `child` rather than levelTwoItem. I also agree with Stowey about using `isActive` on the child rather than adding and removing the whole child property from the object.

Reply:
Yeah, great thoughts. I wrestled with this one for a bit and started out with an array and an `isActive` key but changed to a simpler object, mainly because having one rendered level two item at a time was part of the requirements.

During a conversation with @stowball we decided to switch to using an `activePath` prop on the parent component because it:
1. Creates consistency with our other navigation list components - `MainNav`, `SubNav`, `SideNav`
2. Ensures only active "leaf node"
3. Allows showing an active hierarchy of levelled navigation items
4. Is easier for consumers to use